test/export/test_export.py

```diff
@@ -1773,6 +1774,40 @@ def forward(self, x):
         ):
             export(M(), (torch.randn(2, 3),), strict=False)
 
+    @testing.expectedFailureTrainingIRToRunDecomp  # Could not guard on data-dependent expression -u0 > 16 (unhinted: -u0 > 16)
```

Comment:
This is the first issue in https://github.com/pytorch/pytorch/issues/158366. Need a separate fix.